src/repositories/powerBalanceSnapshotRepository.ts

```diff
@@ -0,0 +1,9 @@
+import { PowerBalanceSnapshot } from '../entities/powerBalanceSnapshot';
+
+export const createPowerSnapshotBalances = async (params: {
+  powerSnapshotId: number;
+  userId: number;
+  balance: number;
+}): Promise<PowerBalanceSnapshot> => {
+  return PowerBalanceSnapshot.create(params).save();
```

Comment:
This `create` function has another overloading that accepts an array of entity-like objects. That maybe more useful for our usecase